packages/idempotency/src/middleware/makeHandlerIdempotent.ts

```diff
@@ -90,6 +90,11 @@ const shouldSkipIdempotency = (request: MiddyLikeRequest): boolean => {
  * ).use(makeHandlerIdempotent({ persistenceStore: dynamoDBPersistenceLayer }));
  * ```
  *
+ * Note that for the middleware to work, your Lambda function handler must return a value different from `undefined`.
+ * This is a [known limitation of Middy.js early return feature](https://github.com/middyjs/middy/issues/1236).
+ *
+ * If your use case requires early returns, you can use the {@link index.makeIdempotent | makeIdempotent()} function wrapper instead.
```

Comment:
If you accept the first suggestion, you need to change it here. If not, it's good.